packages/web/src/hooks/pool/data/use-reposition-handle.tsx

```diff
@@ -267,8 +271,8 @@ export const useRepositionHandle = () => {
       selectPool.maxPrice,
       tickToPrice(ordered ? selectedPosition.tickLower : selectedPosition.tickUpper * -1),
       tickToPrice(ordered ? selectedPosition.tickUpper : selectedPosition.tickLower * -1),
-      selectedPosition.tokenABalance,
-      selectedPosition.tokenBBalance,
+      String(makeDisplayTokenAmount(tokenA, selectedPosition.tokenABalance)),
+      String(makeDisplayTokenAmount(tokenB, selectedPosition.tokenBBalance)),
```

Comment:
this conversion seems unnecessary

```suggestion
      makeDisplayTokenAmount(tokenA, selectedPosition.tokenABalance),
      makeDisplayTokenAmount(tokenB, selectedPosition.tokenBBalance),
```